tests/TestCase/ORM/TableRegistryTest.php

```diff
@@ -320,6 +321,41 @@ public function testConfigAndBuild()
         $this->assertEquals('users', $table->alias());
         $this->assertSame($connection, $table->connection());
         $this->assertEquals(array_keys($schema), $table->schema()->columns());
+        $this->assertEquals($schema['id']['type'], $table->schema()->column('id')['type']);
+    }
+
+    /**
+     * Tests that table options can be pre-configured with a single validator
+     *
+     * @return void
+     */
+    public function testConfigWithSingleValidator()
+    {
+        TableRegistry::config('users', ['validator' => new Validator()]);
+        $table = TableRegistry::get('users');
+
+        $this->assertInstanceOf('Cake\Validation\Validator', $table->validator('default'));
+    }
+
+    /**
+     * Tests that table options can be pre-configured with multiple validators
+     *
+     * @return void
+     */
+    public function testConfigWithMultipleValidators()
+    {
+        TableRegistry::config('users', [
+            'validator' => [
+                'default' => new Validator(),
+                'secondary' => new Validator(),
+                'tertiary' => new Validator(),
+            ]
+        ]);
+        $table = TableRegistry::get('users');
+
+        $this->assertInstanceOf('Cake\Validation\Validator', $table->validator('default'));
+        $this->assertInstanceOf('Cake\Validation\Validator', $table->validator('secondary'));
+        $this->assertInstanceOf('Cake\Validation\Validator', $table->validator('tertiary'));
```

Comment:
You could the same values were returned instead. Right now any validator could be returned and your test would pass.